README-pt-BR.md

```diff
@@ -0,0 +1,347 @@
+| Sem Grandes Arquivos | [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) |
+| Sem "_Lorem Ipsum_"s | [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
+
+O _The Turing Way_ é um guia levemente obstinado para a ciência de dados reproduzível.
+Você pode lê-lo aqui: <https://the-turing-way.netlify.com>
+Neste momento, você está visualizando o repositório GitHub do projeto onde se encontra cada pedacinho que compõe o guia e onde o processo de escrita/construção do guia acontece.
+
+Nosso objetivo é prover todas as informações que os pesquisadores precisam no início de seus projetos para assegurar que eles serão facilmente reproduzíveis quando terminados.
+
+Isso também significa garantir que os alunos de doutorado, pós-doutorandos, pesquisadores principais e equipes de financiamento saibam quais partes da "responsabilidade de reprodutibilidade" podem afetar e o que devem fazer para estimular a ciência de dados a ser mais eficiente, eficaz e compreensível.
+
+Tabela de conteúdos:
+
+- [Sobre o projeto](#sobre-o-projeto)
+- [O time](#o-time)
+- [Como contribuir](#como-contribuir)
+- [Citando o _The Turing Way_](#citando-o-the-turing-way)
+- [Entre em contato](#entre-em-contato)
+- [Contribuidores](#contribuidores)
+
+🎧 Se você preferir uma introdução audível ao projeto, Rachael, que é membra do nosso time, fez uma apresentação no [Open Science Fair 2019](https://www.opensciencefair.eu/) em Porto e sua demo foi gravada pelo [Orion podcast](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
+A visão geral do _The Turing Way_ começa em decorridos 5:13.
+
+### Sobre o projeto
+
+A pesquisa reproduzível é necessária para garantir que trabalhos científicos sejam confiáveis.
+Financiadores e editores estão começando a exigir que as publicações incluam o acesso aos dados subjacentes e ao código de análise.
+O objetivo é garantir que todos os resultados possam ser verificados de forma independente e aproveitados em trabalhos futuros.
+Às vezes, é mais fácil falar do que fazer.
+Compartilhar esses resultados de pesquisa significa compreender o gerenciamento de dados, o funcionamento das bibliotecas, o desenvolvimento de software e as técnicas de integração contínua: habilidades que não são, de forma ampla, ensinadas ou esperadas de pesquisadores acadêmicos e cientistas de dados.
+Como essas atividades não são comumente ensinadas, reconhecemos que a carga da exigência e a aquisição de novas habilidades podem ser intimidantes para pessoas que são novas neste mundo.
```

Comment:
Como essas atividades não são normalmente ensinadas, reconhecemos que a carga da exigência e a aquisição de novas habilidades podem ser intimidantes para pessoas que são novas neste mundo.

Reply:
I honestly believe *comumente* is a better choice for this specific case.